src/super_gradients/recipes/arch_params/yolox_l_arch_params.yaml

```diff
@@ -0,0 +1,12 @@
+defaults:
+  - yolo_arch_params
+
+anchors:
+  _target_: super_gradients.training.utils.detection_utils.Anchors
+  anchors_list: [[], [], []]
```

Comment:
Does this work in DDP? If you define them here I think they won't be overridden and training will crash on syncing empty shapes. Not sure about overriding order though

Reply:
We always override them. Not a problem.

Reply:
You set them manually (like in #270) and only then override with what's coming from arch_params. So if arch_params contain empty anchors that's what will be sed in a model and it'll crash

Reply:
changed all anchors to be filled with 0s Added to callbacks factory + factory call in SG in https://github.com/Deci-AI/super-gradients/commit/733237b8f40c7d6fbb911edc67d36201d0c131af